collectors/channels_collector.go

```diff
@@ -174,10 +184,45 @@ func NewChannelsCollector(lnd lndclient.LightningClient, errChan chan<- error,
 			[]string{"amount"}, nil,
 		),
 
-		lnd:         lnd,
-		primaryNode: cfg.PrimaryNode,
-		errChan:     errChan,
+		lnd:                 lnd,
+		primaryNode:         cfg.PrimaryNode,
+		closedChannelsCache: nil,
+		errChan:             errChan,
+		quit:                quitChan,
 	}
+
+	// Perform an initial refresh for the cache
+	collector.refreshClosedChannelsCache()
+
+	// Start a ticker to update the cache once per 10m
+	go func() {
+		ticker := time.NewTicker(10 * time.Minute)
```

Comment:
Should we make this a package-level variable just so "magic numbers" are easier to find and change?

Reply:
The 10 minute magic number is still here in the latest push.